### PR TITLE
adjusted frontend to reflect changes to entity file

### PR DESCRIPTION
**Pull request by pabucsb**
Please merge #19 first.

In this PR I adjusted the frontend in order to reflect the change made to one of my entity variables. 
Link to storybook: https://ucsb-cs156-m23.github.io/proj-gauchoride-m23-9am-1/

image

image

### CODE REVIEW

**Comment by pconrad** [comment]
@pabucsb Please make the title something descriptive.   

For example: `Add and Rename some fields in Ride entity`

**Review by alecmorrison — APPROVED** [review]
Looks good, other than committing the previous changes

File: frontend/src/main/components/Ride/RideTable.js [hunk 1]
```diff
@@ -67,9 +67,19 @@ export default function RideTable({
...
+        // {
+        //     Header: 'Pick Up Room',
+        //     accessor: 'pickupRoom',
+
+        // },
+        // {
+        //     Header: 'Notes',
+        //     accessor: 'notes',
+
+        // }
```
Comment (pconrad) [inline frontend/src/main/components/Ride/RideTable.js:82]: Why is this commented out?

File: frontend/src/main/components/Ride/RideTable.js [hunk 2]
```diff
@@ -106,9 +116,19 @@ export default function RideTable({
...
```
Comment (pconrad) [inline frontend/src/main/components/Ride/RideTable.js:131]: Why is this commented out?

File: frontend/src/main/components/Ride/RideTable.js [hunk 3]
```diff
@@ -145,9 +165,19 @@ export default function RideTable({
...
```
Comment (pconrad) [inline frontend/src/main/components/Ride/RideTable.js:180]: Why are these commented out?

File: src/main/java/edu/ucsb/cs156/gauchoride/controllers/RideController.java [hunk 4]
```diff
@@ -90,25 +90,35 @@ public Ride postRide(
...
+        @Parameter(name="notes", description="String, Notes for additional information", example="Library entrance by the arbor", required = false)
+        @RequestParam String notes
+
+
         )
         {
 
         Ride ride = new Ride();
-        
+        //not sure if we have to set dropoffroom and pickuproom
```
Comment (pconrad) [inline src/main/java/edu/ucsb/cs156/gauchoride/controllers/RideController.java:110]: See feedback on PR #19

**Review by pconrad — CHANGES_REQUESTED** [review]
Help me understand why the commented out code is there?

File: frontend/src/main/components/Ride/RideForm.js [hunk 5]
```diff
@@ -147,21 +147,22 @@ function RideForm({ initialContents, submitAction, buttonLabel = "Create" }) {
...
-                    {...register("room", {
-                        required: "Room number is required."
-                    })}
-                    placeholder="e.g. 2225"  
-                    defaultValue={initialContents?.room} 
+                    {...register("dropoffRoom")}
+                    isInvalid={Boolean(errors.dropoffRoom)}
+                    //{...register("dropoffRoom", {
+                        //required: "Drop OffRoom number is required."
+                    //})}
```
Comment (Aasish-Virjala) [inline frontend/src/main/components/Ride/RideForm.js:159]: Remove comment
```suggestion
```

**Review by Aasish-Virjala — DISMISSED** [review]
Remove comments, then it looks good.

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**